### PR TITLE
Get involved map

### PR DESCRIPTION
**Pull request by alstnc**
Added the map to /getinvolved using Mapbox. Marker isn't currently working, need to figure that out.

Dependencies:
"mapbox-gl"
"react-mapbox-gl"

### CODE REVIEW

File: src/components/map.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+"use client";
+import React from 'react';
+import ReactMapboxGl, { Layer, Feature } from 'react-mapbox-gl';
+import 'mapbox-gl/dist/mapbox-gl.css';
+
+const Map = ReactMapboxGl({
+  accessToken: 'pk.eyJ1IjoidGhlZ2l2aW5naGVhcnQiLCJhIjoiY2xwNWl3bm50MWVudzJqbzRzMHQ4cHRpdyJ9.hd8bhxh4uUXhe1xA1X9tCA'
```
Comment (codingdn) [inline src/components/map.tsx:7]: should this be public or put into the environment variables?

**Review by codingdn — CHANGES_REQUESTED** [review]

File: src/components/map.tsx (hunk 1 above)
Comment (alstnc) [reply src/components/map.tsx]: It was in .env.local initially but I thought it was the generic public one so it was fine. Now realizing that's prob not the case, I've put it back

Comment (alstnc) [reply src/components/map.tsx]: I've added the env variables to the resources section of Notion

**Review by codingdn — APPROVED** [review]
lgtm

**Pull request merged**